tests/util/gentle-register-property.js

```diff
@@ -6,15 +6,16 @@
  * @param {string} [meta.syntax] - Property syntax.
  * @param {boolean} [meta.inherits] - Whether the property inherits.
  * @param {*} [meta.initialValue] - Initial value.
+ * @param {Window} [window] - The window to check in.
```

Comment:
We probably need to update the description here as well to align with the main utility function.

Reply:
Done in https://github.com/LeaVerou/style-observer/pull/111/commits/9e0a3540b3d7435c6596fde0669e1fd1c84a2317